example/app.py

```diff
@@ -6,10 +6,21 @@
 PORT = 5000
 app = Flask(__name__)
 
-app.config.update({'SERVER_NAME': 'localhost:{}'.format(PORT),
-                   'SECRET_KEY': 'dev_key'})
-auth = OIDCAuthentication(app, issuer="https://localhost:50009")
-
+# See http://flask.pocoo.org/docs/0.12/config/
+app.config.update({'SERVER_NAME': 'example.com',
+                   'SECRET_KEY': 'dev_key',
+                   'PREFERRED_URL_SCHEME': 'https',
+                   'SESSION_PERMANENT': True, #turn on flask session support
```

Comment:
@andrewkrug any idea why this doesn't stick and the original author has to hardcode it?
Its also hard coded at #L128 now